network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/chart_section.html

```diff
@@ -0,0 +1,55 @@
+{% load static i18n %}
+<section class="tw-flex tw-flex-col tw-justify-center tw-items-center tw-my-7 medium:tw-my-[80px]">
+  <img src="{% static "_images/youtube-regrets/regrets-reporter/our-research-desktop.png" %}" alt="">
+  <div class="container tw-mt-5 medium:tw-mt-7 tw-space-y-7 medium:tw-space-y-[80px]">
+    <div class="tw-text-center tw-max-w-3xl tw-mx-auto">
+      {% blocktrans trimmed %}
+        <h2 class="tw-text-4xl medium:tw-text-[3.25rem] tw-leading-none">The result? People can’t escape bad recommendations.</h2>
+        <p>In the quantitative portion of our study, we ran a randomized controlled experiment across our community of RegretsReporter participants that could directly test the effectiveness of YouTube’s user controls. We found that YouTube’s user controls somewhat influence what is recommended, but this effect is meager and most unwanted videos still slip through.</p>
+      {% endblocktrans %}
+    </div>
+
+    {# Chart #}
+    <div class="tw-max-w-4xl tw-mx-auto tw-text-center">
+      <div class="tw-mb-5">
+        {% blocktrans trimmed %}
+          <p class="tw-uppercase tw-text-2xl tw-font-bold tw-mb-0">Percentage of Bad Recommendations</p>
+          <p class="tw-italic tw-text-xl">among all video pairs assessed by our research assistants</p>
+        {% endblocktrans %}
+      </div>
+      <iframe aria-label="{% trans 'YouTube Percentage of Bad Recommendations Chart' %}"
+              class="tw-w-0 tw-min-w-full tw-border-none tw-h-[140px]"
+              src="https://datawrapper.dwcdn.net/kl3jI/7/">
```

Comment:
Great question, i've asked inside of the youtube channel to clarify this. However maybe it can be something we tackle in a future ticket so this doesn't get held up :)